draft-irtf-nmrg-network-digital-twin-arch.md

```diff
@@ -423,37 +423,38 @@ Digital Twin Network Online Session {{DTPI2021}}, {{DTPI2022}}, and IEEE
 NOMS 2022 - TNT workshop {{TNT2022}}.
 
 Although the application of digital twin technology in networking has
-started, the research of digital twin for networks technology is
+started, the research on digital twins for networks technology is
 still in its infancy.  Current applications focus on specific
-scenarios (such as network optimization), where Network Digital Twin
-is just used as a network simulation tool to solve the problem of
+scenarios (such as network optimization), where a Network Digital Twin
+is used as a network simulation tool to solve particular problems in
 network operation and maintenance.  Combined with the characteristics
 of digital twin technology and its application in other industries,
 this document believes that Network Digital Twin can be regarded as
-an indispensable part of the overall network system and provides a
-general architecture involving the whole life cycle of a "real"
-(typically, physical) network in the future, serving the application
-of network innovative technologies such as network planning, construction,
-maintenance and optimization, improving the automation and intelligence
-level of the network.
+an indispensable part of the overall network system, and can play an
+important role generally in architectures serving use cases across the
+whole life cycle of a "real" (typically, physical) network. Such use cases
+and applications span the range of network operations such as network
+planning, construction, maintenance and optimization, and improve the
+automation and intelligence level of the network.
```

Comment:
```suggestion
automation and intelligence level of the network).
```

Reply:
Med, Can you check again whether this right 
parentheses is needed？